scripts/create-package.js

```diff
@@ -0,0 +1,183 @@
+// Dependencies
```

Comment:
I have found vscode helps catch some silly issues if you add:
```
// @ts-check
```

Reply:
Adding this to the file resulted in some errors that seem irrelevant, like not being able to require mustache or argv.